docker-compose-runner.yml

```diff
@@ -6,8 +6,13 @@ services:
         - POSTGRES_HOST_AUTH_METHOD=trust
 
   rskj:
-    build:
-      context: ./docker/rskj
+    image: rsksmart/rskj:HOP-4.4.0
+    ports:
+      - "4444:4444"
+      - "4445:4445"
+    volumes:
+      - ./docker/rskj/logback.xml:/etc/rsk/logback.xml
+      - ./docker/rskj/node.conf:/etc/rsk/node.conf
```

Comment:
@patogallaiovlabs  I've done it but not for the runner, because I thought that it wouldn't make sense for the runner to have it, while it can be very valuable for the dev env